node_modules/col-assets/lib/api.js

```diff
@@ -1035,13 +1038,13 @@ var generatePreviews = module.exports.generatePreviews = function(asset, callbac
   };
 
   var assetUrl = null;
-  if (asset.type === 'file') {
+  if (asset.type === 'file' || asset.type === 'whiteboard') {
     assetUrl = asset.download_url;
   } else if (asset.type === 'link') {
     assetUrl = asset.url;
-  } else {
-    // Don't generate previews for whiteboards, the Whiteboards API
-    // takes care of this
```

Comment:
Part 1 of the bug: the assets API assumed that the whiteboards API would handle previews for exported whiteboards.